src/entity/Feed.ts

```diff
@@ -8,4 +15,28 @@ export class Feed {
   @Column({ type: 'text' })
   @Index()
   userId: string;
+
+  @Column({ default: () => 'now()', update: false })
+  createdAt: Date;
+
+  @Column({ type: 'jsonb', default: {} })
+  flags: FeedFlags = {};
+
+  @Column({
+    type: 'text',
+    update: false,
+    insert: false,
+    nullable: false,
+    unique: true,
+    generatedType: 'STORED',
+    asExpression: `trim(BOTH '-' FROM regexp_replace(lower(trim(COALESCE(LEFT(feed.flags->>'name',100),'')||'-'||feed.id)), '[^a-z0-9-]+', '-', 'gi'))`,
+  })
+  @Index('IDX_feed_slug', { unique: true })
+  slug: string;
+
+  @ManyToOne(() => User, {
+    lazy: true,
+    onDelete: 'CASCADE',
+  })
+  user: Promise<User>;
```

Comment:
We have `userId` already so we don't need `createdBy` from RFC I think.

Reply:
Is it possible that two users can share the same feed ID?
In that case, it would probably be useful to have createdBy, as that's the person who can delete a feed?

<table>
<tr>
 <td>Feed ID
 <td>User ID
<tr>
 <td>feed123
 <td>user1
<tr>
 <td>feed123
 <td>user2
</table>

Reply:
No, `id` is PK and unique in `feed` table.